neqo-transport/src/frame.rs

```diff
@@ -103,7 +103,9 @@ pub struct AckRange {
 
 #[derive(PartialEq, Eq, Debug, Clone)]
 pub enum Frame<'a> {
-    Padding,
+    Padding {
+        length: u64,
+    },
```

Comment:
Padding(u64) would be OK.  But I question the need for a u64 here.  Wouldn't u16 work?